spec/support/api/schemas/song.json

```diff
@@ -76,6 +76,10 @@
         "formatted_duration": {
           "id": "http://spotify-apprentice-app.com/song/title",
           "type": "string"
+        },
+        "genre_id": {
+          "id": "http://spotify-apprentice-app.com/song/title",
+          "type": "string"
```

Comment:
Also you want the `id` property to reflect the name of the property.  So instead of:

`"id": "http://spotify-apprentice-app.com/song/title"`

you want something like:

`"id": "http://spotify-apprentice-app.com/song/genre_id"`